### ISSUE TITLE
Accordion content section has Axe regression

### ISSUE THREAD

**Issue by colinbruce** [alphagov/govuk-frontend#4298]
## Description of the issue

The accordion content section recommends aria-labelledby
https://github.com/alphagov/govuk-frontend/blob/f837f9c003a50f9e64a2fe3476be91e6e228f318/packages/govuk-frontend/src/govuk/components/accordion/template.njk#L54

Our application runs automated tests using the [axe-core](https://github.com/dequelabs/axe-core-gems) suite of gems
With the update to 4.8.0 it has started raising errors stating:
```aria-labelledby attribute cannot be used on a div with no valid role attribute```

Changing the recommended code to 
```html
<div id="{{ id }}-content-{{ loop.index }}" role="region" class="govuk-accordion__section-content" aria-labelledby="{{ id }}-heading-{{ loop.index }}"> 
```

Allows the tests to pass, but will probably need accessibility guidance on the correct role to assign 

This was raised with the [x-govuk/govuk-components](https://github.com/x-govuk/govuk-components/issues/452) team and is raised here as this template is the source for their work

## Steps to reproduce the issue

Run the 4.8.0 Axe gems against html generated using the suggested pattern 

## Actual vs expected behaviour

Warnings vs no warnings! 😄 

## Environment (where applicable)

- Operating system: OSX 13,5,2
- Browser: Chrome for testing
- Browser version: 117
- GOV.UK Frontend Version: 4.6.0

**Comment by 36degrees** [comment]
Hey @colinbruce, thanks for raising this.

I think this might be a duplicate of #2472 – could you take a look and see if you think that issue covers it?

Thanks!

**Comment by colinbruce** [comment]
Apologies, I read the initial post but not the followup discussion!

Where that ticket has moved to does seem to cover this yes :+1:

Should I close this and reference that issue?

**Comment by 36degrees** [comment]
If it's alright with you I'll close this as a duplicate.

Feel free to add a comment on #2472 if you have anything to add – I don't think the specific error message you're seeing has been mentioned there, so it might be useful to record that in case others are searching for it.

We're busy working on the next major release at the minute but I'm hoping we'll be able to resolve this issue soon. As far as I know although it's flagged by Axe we're not aware of any problems it would cause users, but it has come up quite a lot so it'd be good to get it fixed anyway!
